src/services/cloudinary.js

```diff
@@ -0,0 +1,22 @@
+import {v2 as cloudinary,} from "cloudinary"
+import  dotenv  from 'dotenv';
+
+dotenv.config()
```

Comment:
suggestion: I think you can call this only once in the main file, or build a configService to centralize all env variables